Specs/CreateAccountSuccess.js

```diff
@@ -0,0 +1,17 @@
+
+let LandingPage = require('../PageObjects/LandingPage');
+let LoginPage = require('../PageObjects/LoginPage');
+let RegistrationPage = require('../PageObjects/RegistrationPage');
+let RegistrationFinalPage = require('../PageObjects/RegistrationFinalPage');
+
+describe ('CreateAccountSuccess', () => {
+    it('CreatingAccount', async () => {
+        await LandingPage.navigateToLogin(browser.params.url.urla);
+        await LoginPage.waitForLoginPageLoaded();
+        await LoginPage.clickRegisterButton();
+        await RegistrationPage.waitForRegistrationPageLoaded();
+        await RegistrationPage.RegisterUser(browser.params.register.email, browser.params.register.nickname, browser.params.register.pass);
```

Comment:
You can't use always same account for account registration (it will fail as already registered).
Update this test to use always pseudo new email.
Use following workaround:
`jimbo2382+<current time in milliseconds>@gmail.com`

Reply:
Done with SuccessfulLogin2